underlay/src/main/resources/config/datamapping/aouCT_testonly/entity/variant/entity.json

```diff
@@ -2,21 +2,21 @@
   "name": "variant",
   "allInstancesSqlFile": "all.sql",
   "attributes": [
-    { "name": "id", "dataType": "INT64", "valueFieldName": "row_num" },
-    { "name": "variant_id", "dataType": "STRING", "valueFieldName": "vid" },
+    { "name": "id", "dataType": "STRING", "valueFieldName": "vid" },
     { "name": "gene", "dataType": "STRING", "valueFieldName": "gene_symbol", "isComputeDisplayHint": true },
     { "name": "rs_number", "dataType": "STRING", "isDataTypeRepeated": true, "valueFieldName": "dbsnp_rsid" },
-    { "name": "genomic_location", "dataType": "STRING" },
     { "name": "consequence", "dataType": "STRING", "isDataTypeRepeated":  true, "isComputeDisplayHint": true },
     { "name": "protein_change", "dataType": "STRING", "valueFieldName": "aa_change" },
     { "name": "clinvar_significance", "dataType": "STRING", "isDataTypeRepeated":  true, "valueFieldName": "clinvar_classification", "isComputeDisplayHint": true },
     { "name": "allele_count", "dataType": "INT64", "valueFieldName": "gvs_all_ac", "isComputeDisplayHint": true },
     { "name": "allele_number", "dataType": "INT64", "valueFieldName": "gvs_all_an", "isComputeDisplayHint": true },
-    { "name": "allele_frequency", "dataType": "DOUBLE", "valueFieldName": "gvs_all_af", "isComputeDisplayHint": true }
+    { "name": "allele_frequency", "dataType": "DOUBLE", "valueFieldName": "gvs_all_af", "isComputeDisplayHint": true },
+    { "name": "contig", "dataType": "STRING" },
+    { "name": "position", "dataType": "INT64" }
   ],
   "idAttribute": "id",
   "textSearch": {
-    "attributes": [ "variant_id", "gene", "rs_number", "genomic_location" ]
+    "attributes": [ "gene", "rs_number", "contig", "position" ]
```

Comment:
text search will be replaced with clustered tabled in https://github.com/DataBiosphere/tanagra/pull/1098